AppOnboard/app/src/main/java/com/example/taqtile/apponboard/User.java

```diff
@@ -0,0 +1,44 @@
+package com.example.taqtile.apponboard;
+
+/**
+ * Created by taqtile on 5/2/16.
+ */
+public class User {
+    int id;
+    String first_name;
+    String last_name;
+    String avatar;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getFirst_name() {
```

Comment:
Em Java, por convenção, nós utilizamos o camelCase (https://developers.google.com/android/for-all/vocab-words/) para representar variáveis com 2 nomes ou mais.